portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,63 @@
+function makeQuiz(quizContainer, questions) {
+  let output = [];
+  questions.forEach((currentQuestion, questionNumber) => {
+    output.push(
```

Comment:
Create individual DOM nodes and add them as children to quizContainer. Setting innerHTML can expose you to security vulnerabilities.